### DISCUSSION TITLE
Question about the free tier and commercial use

### DISCUSSION THREAD

**Discussion by charichu** [vercel/community#5568] [Feature Question] [answered]
### Question

Hello,

Since I read here, that hosting a portfolio site is not commercial, I was wondering If I can use the free tier to create a site about a small pen&paper RPG streaming group. The Site will just contain informations about the Players/Characters and world

**Comment by amyegan** [comment, marked as answer]
In general, if you're not making money from the site, not exceeding limits, and not doing anything malicious or illegal, then the site should be fine. But I recommend reviewing our [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#) and [Terms of Service](https://vercel.com/legal/terms#fair-use) to be sure for your exact use case.
